tests/components/ffmpeg/test_sensor.py

```diff
@@ -30,21 +30,23 @@ def test_setup_component(self):
         assert self.hass.data["ffmpeg"].binary == "ffmpeg"
         assert self.hass.states.get("binary_sensor.ffmpeg_noise") is not None
 
-    @patch("haffmpeg.sensor.SensorNoise.open_sensor", return_value=mock_coro())
-    def test_setup_component_start(self, mock_start):
+    async def test_setup_component_start(self):
```

Comment:
I suggest we rewrite the tests to be standalone async test functions so we can use our pytest fixtures.

I'm not sure this test is working properly when async and calling non async test infrastructure.